android/src/com/frostwire/android/gui/fragments/preference/OtherFragment.java

```diff
@@ -110,7 +110,7 @@ private void setupClearIndex() {
                 @Override
                 public void onClick(View v) {
                     LocalSearchEngine.instance().clearCache();
-                    UIUtils.showShortMessage(getActivity(), R.string.deleted_crawl_cache);
```

Comment:
Why don't we just make the old `UIUtils.showShortMessage` use the material style if it can, otherwise fall back to non styled toasts?

CC: @marcelinkaaa, I guess I just want to know if you want to now style all toasts, if so, let's do what I said above, and avoid having 2 methods for the same purpose.